pkg/printers/json.go

```diff
@@ -8,22 +8,30 @@ import (
 	"github.com/golangci/golangci-lint/pkg/result"
 )
 
-type JSON struct{}
+type JSON struct {
+	disableCongrats bool
```

Comment:
JSON knows nothing about congrats, `silent` or more specific for JSON like `omitEmpty`

Reply:
My vote is for silent here as well I think.